apps/antalmanac/src/components/Calendar/CalendarRoot.tsx

```diff
@@ -239,6 +239,9 @@ export default function ScheduleCalendar(props: ScheduleCalendarProps) {
                     views={[Views.WEEK, Views.WORK_WEEK]}
                     defaultView={Views.WORK_WEEK}
                     view={hasWeekendCourse ? Views.WEEK : Views.WORK_WEEK}
+                    onView={() => {
+                        return;
+                    }}
```

Comment:
Since we're now in this file, I'm taking another liberty to clean up this console error